src/components/ContactImportation/ContactImportationCard.jsx

```diff
@@ -0,0 +1,196 @@
+        );
+
+      case Status.FILE_ISSUE:
+        return (
+          <CardWrapper clickable={true}>
+            <ImportationFile
+              icon={IconFileWrongFormat}
+              name={importation.file.name}
+            />
+            <span className="importation-message">
+              {t(importation.fileIssue)}
+            </span>
+            <TransferButton fileAction={onFileSelected} />
+          </CardWrapper>
+        );
+
+      case Status.READY:
+        return (
+          <CardWrapper clickable={false}>
+            <ImportationFile
+              icon={IconFileVcf}
+              name={importation.file.name}
+              unselectAction={onFileUnselected}
+            />
+            <span className="importation-status">{t("importation.ready")}</span>
+          </CardWrapper>
+        );
+
+      case Status.RUNNING:
+        return (
+          <CardWrapper clickable={false}>
+            <ImportationFile icon={IconFileVcf} name={importation.file.name} />
+            <span className="importation-status">
+              {t("importation.running")}
+            </span>
+            {progress && (
+              <progress value={progress.current} max={progress.total} />
+            )}
+          </CardWrapper>
+        );
+
+      case Status.PARTIAL_SUCCESS:
+        return (
+          <CardWrapper clickable={true}>
+            <ImportationFile
+              icon={IconFilePartialImport}
+              name={importation.file.name}
+              unselectAction={onFileUnselected}
+            />
+            <span className="importation-message">
+              {t("importation.partial_success", {
+                smart_count: report.imported,
+                total: report.total
+              })}
+            </span>
+            {Importation.canRetry(importation) && (
+              <span className="importation-message">
+                {t("importation.retry_hint")}
+              </span>
+            )}
+          </CardWrapper>
+        );
+
+      case Status.COMPLETE_FAILURE:
+        return (
+          <CardWrapper clickable={true}>
+            <ImportationFile
+              icon={IconFileFailImport}
+              name={importation.file.name}
+            />
+            <span className="importation-message">
+              {t("importation.complete_failure")}
+            </span>
+            {Importation.canRetry(importation) && (
+              <span className="importation-message">
+                {t("importation.retry_or_select_another_hint")}
+              </span>
+            )}
+            <TransferButton fileAction={onFileSelected} />
+          </CardWrapper>
+        );
+
+      default:
+        // eslint-disable-next-line
+        console.warn(
+          "ContactImportationCard: unexpected importation status:",
+          importation.status
+        );
+    }
+  }
+}
+ContactImportationCard.propTypes = {
+  importation: Importation.propType.isRequired,
+  progress: PropTypes.object,
```

Comment:
Non required `props` should have default value.

Reply:
Oh, ok. Even when the default value is actually `undefined`?

Reply:
at least it is explicit, I guess.

Reply:
👌